kernel/packages/shared/apis/SceneStateStorageController/BuilderServerAPIManager.ts

```diff
@@ -136,6 +136,10 @@ export class BuilderServerAPIManager {
   }
 
   private async setManifestOnServer(builderManifest: BuilderManifest, identity: ExplorerIdentity) {
+
+    //TODO: We should delete this when we enter in production or we won't be able to set the project in production
+    if (getDefaultTLD() === 'org')
```

Comment:
```suggestion
    if (getDefaultTLD() === 'org')
```

Maybe we can put a friendly log message just in case this get misinterpreted as a bug?

Reply:
Done!